system-addon/content-src/components/Search/_Search.scss

```diff
@@ -60,7 +64,17 @@
 
     &:focus,
     &:hover {
-      background-color: $grey-90-10;
+      &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        offset-inline-start: 0;
+        width: 100%;
+        height: 100%;
```

Comment:
nit: Might as well just do top/right/bottom/left:0 in case it seems like this is rtl-special.